docs/user-guide/widgets.md

```diff
@@ -214,35 +214,35 @@ An example of counter widget could be:
 
-* Move and resize them
+<img src="../img/widgets/widget-button.jpg" class="ms-docimage" style="max-width:500px;"/>
 
-* Fix and expand/collapse them
-
-* Access their menu
-
-### Move and resize a widget
-
-In order to move a widget you can simply drag and drop it inside the viewport, and in order to resize it you can click in the lower right corner and drag it to the desired size:
+* Drag and drop the widget to move it within the viewer and **resize** it thought the <img src="../img/button/resize-button.jpg" class="ms-docbutton"/> button (also available for widgets present in a dashboard)
 
 <img src="../img/widgets/ded-widgets.gif" class="ms-docimage" style="max-width:500px;"/>
 
-### Fix and expand/collapse a widget
+* **Fix** the position and the dimension of the widget thought the <img src="../img/button/pin.jpg" class="ms-docbutton"/> button
+
+* **Collapse** the widget thought the <img src="../img/button/collapse-button.jpg" class="ms-docbutton"/> button and expand it by clicking its icon in the *Widgets Tray*
 
-The position and the dimension of a widget can be fixed with a click on the **Pin** button <img src="../img/button/pin.jpg" class="ms-docbutton"/>. <br>
-The *Widgets Tray*, otherwise, allows the user to expand/collapse each single widget individually <img src="../img/button/wid-tray-single.jpg" class="ms-docbutton"/> or all of them at the same time <img src="../img/button/wid-tray-all.jpg" class="ms-docbutton"/>. A single widget can be also collapsed through the **Collapse** button <img src="../img/button/collapse2.jpg" class="ms-docbutton"/>.
+<img src="../img/widgets/collapse-widgets.gif" class="ms-docimage" style="max-width:500px;"/>
 
 !!!note
+    The *Widgets Tray*, otherwise, allows the user to expand/collapse each single widget individually <img src="../img/button/wid-tray-single.jpg" class="ms-docbutton"/> or all of them at the same time <img src="../img/button/wid-tray-all.jpg" class="ms-docbutton"/>.
+
+!!!warning
     When both **[Timeline](timeline.md)** and widgets are present in a map, the *Timeline* button appears in the *Widgets Tray* <img src="../img/button/w-tray-timeline.jpg" class="ms-docbutton"/> allowing the user to expand and collapse it (widgets and *Timeline* can't anyhow be expanded at the same time).
 
-### Access widgets info
+* Make the widget **Full screen** thought the <img src="../img/button/maximize-button.jpg" class="ms-docbutton"/> button (also available for widgets present in a dashboard)
+
+<img src="../img/widgets/maximize-widgets.gif" class="ms-docimage" style="max-width:500px;"/>
 
-As soon as a description is provided for a widget the info button <img src="../img/button/info.jpg" class="ms-docbutton"/> appears, allowing the user to take a look at the widget's *Title* and *Description*:
+* Access to the *Title* and *Description* info thought the <img src="../img/button/info.jpg" class="ms-docbutton"/> button, if this information was provided for the widget
```

Comment:
* Access to the *Title* and *Description* info thought the <img src="../img/button/info.jpg" class="ms-docbutton"/> button, if this information have been provided during the widget configuration/creation.